.github/containerscan/allowedlist.yaml

```diff
@@ -1,3 +1,4 @@
 general:
   vulnerabilities:
-  - CVE-2022-2097 # as I read it https://github.com/advisories/GHSA-3wx7-46ch-7rq2 shouldn't affect VAPE
\ No newline at end of file
+  - CVE-2022-37434
+  - CVE-2021-46828
```

Comment:
I removed a CVE which is fixed in the newer image, unfortunately there are two still unpatched CVEs in the latest (presumably also in the prior image). I looked into the debian-slim base last time, that has even more vulns pop up.
I also checked in on the distroless python image, but that is still not production ready.